### DISCUSSION TITLE
What's the last part of Manifest CR name?

### DISCUSSION THREAD

**Discussion by ukff** [kyma-project/lifecycle-manager#1079] [Q&A] [answered]
In Manifest CR name, what is the last part of name?
for example i have Manifest with name c1301788-e7b3-4fbf-a8f1-249b31fee4c9-btp-operator-4128321918
First part is runtimeID, second is module name and what third part mean(4128321918) and when does it changes ?

**Comment by ruanxin** [comment, marked as answer]
The third part is a hash value based on the fqdn(fully qualified domain name) of the module.

Take template operator as an example, the [fqdn name](https://github.com/kyma-project/template-operator/blob/1a0fc6209460580302e358c053d55d8892342224/module-config.yaml#L1) is what configured in the `module-config.yaml`.

If the FQDN of a module remains unchanged, as it should, the second and third parts of the Manifest CR name for that module will stay constant. The Lifecycle Manager considers it as unique identifier for a module. 

Further detail of the manifest name can be found [here](https://github.com/kyma-project/lifecycle-manager/blob/bdc47384af63b8abfc5c2e9d27600764209a60bc/pkg/module/common/module.go#L91)
